.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[package.json]
+indent_style = space
+indent_size = 2
```

Comment:
this file will be parsed and rewritten by `npm`/`yarn` with 2 spaces. so we would constantly have the whole file as diff.